.github/workflows/build.yaml

```
@@ -187,11 +187,13 @@ jobs:
         pushd build
         cmake .. -DCMAKE_BUILD_TYPE:STRING="${{ matrix.config }}" -DOPTION_BUILD_TRANSLATIONS="ON" -DOPTION_BUILD_WEBSITE_TOOLS="ON" -DOPTION_ASAN="ON" -DOPTION_BUILD_CODECHECK="OFF"
         make -k -j$(nproc) VERBOSE=1
-        TEST_INSTALL_DIR="{$PWD/test_install}"
+    - name: Make install/uninstall
+      run: |
+        cd build
+        TEST_INSTALL_DIR="${PWD}/test_install}"
```

@matthiakl I'm pretty sure the unwanted removals were due to my poor brace placement.

---

Indeed, now it looks better, but you missed one:
```suggestion
        TEST_INSTALL_DIR="${PWD}/test_install"
```